### PR TITLE
[12.0][MIG] mis_builder_operating_unit

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:  LGTM

**Comment by AaronHForgeFlow** [comment]
Could you fix oca dependencies?

**Review by AaronHForgeFlow — APPROVED** [review]
Functinal test good. However, a mis_builder_budget_operating_unit will be needed as well...

**Comment by MiquelRForgeFlow** [comment]
> However, a mis_builder_budget_operating_unit will be needed as well...

In this same PR?

**Comment by AaronHForgeFlow** [comment]
> > However, a mis_builder_budget_operating_unit will be needed as well...
> 
> In this same PR?

No, we will need to migrate this https://github.com/OCA/operating-unit/pull/122

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
Can you please check runbot errors?

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional review

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

File: mis_builder_operating_unit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# © 2015-19 Eficent Business and IT Consulting Services S.L. -
+# Jordi Ballester Alomar
+# © 2015-19 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    'name': 'MIS Builder with Operating Unit',
+    'version': '12.0.1.0.0',
+    'category': 'Reporting',
+    "author": "Eficent Business and IT Consulting Services S.L., "
```
Comment (AdriaGForgeFlow) [inline mis_builder_operating_unit/__manifest__.py:10]: Could you take this opportunity to update this and copyrights as well?

**Pull request merged**